src/category_theory/comma.lean

```diff
@@ -229,6 +230,56 @@ variables {Y : T} {f : X ⟶ Y} {U V : over X} {g : U ⟶ V}
 @[simp] lemma map_map_left : ((map f).map g).left = g.left := rfl
 end
 
+section iterated_slice
+variables (f : over X)
+
+/-- Given f : Y ⟶ X, this is the obvious functor from (T/X)/f to T/Y -/
+@[reducible]
```

Comment:
Try using `@[simps]` rather than `@[reducible]` when defining structures. This automatically add simp lemmas for the structure projections producing data, and won't just unfold the entire thing. Nearly always it's good enough, and results in much less mess.

Reply:
Done, thanks!

Reply:
(For comment threads that don't require further review, can you click the "resolve conversation" button yourself, please?)